cmake/tribits/core/package_arch/TribitsDefineStandardCompileVars.cmake

```diff
@@ -108,14 +108,8 @@ MACRO(TRIBITS_DEFINE_STANDARD_COMPILE_FLAGS_VARS  ENABLE_SHADOWING_WARNINGS)
   IF (${PROJECT_NAME}_CXX_STRONG_COMPILE_WARNING_FLAGS)
     SET(CXX_STRONG_COMPILE_WARNING_FLAGS ${${PROJECT_NAME}_CXX_STRONG_COMPILE_WARNING_FLAGS})
   ELSE()
-    IF (NOT ${PROJECT_NAME}_ENABLE_CXX11)
-       SET(CXX_98_OPTION " -std=c++98") # C++98 standard code
-    ELSE()
-       SET(CXX_98_OPTION)
-    ENDIF()
     MULTILINE_SET(CXX_STRONG_COMPILE_WARNING_FLAGS
       ${COMMON_STRONG_COMPILE_WARNING_FLAGS}
-      "${CXX_98_OPTION}"
```

Comment:
Was dumb to add `-std=c++98` to warning flags.  That was just bad programming on my part :-(